diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is a virtual hub for repositories used to share code, track changes, and let other view projects you've worked on. I'd equivalate it to Google-docs + Dropbox for code.
```

Comment:
Correct. In other words, its a User Interface for Git and version control as well as a 'cloud' for code.